.github/workflows/tests.yml

```diff
@@ -61,17 +63,22 @@ jobs:
         run: vendor/bin/pest
         env:
           DB_CONNECTION: mysql
-          DB_USERNAME: root
+          DB_DATABASE: pulse
+          DB_USERNAME: pulse
+          DB_PASSWORD: password
+          DB_COLLATION: utf8mb4_unicode_ci
 
   mariadb:
     runs-on: ubuntu-22.04
 
     services:
       mysql:
-        image: mariadb:10.5
+        image: mariadb:10
```

Comment:
It seems MariaDB 10.5 doesn't support the new default collation. Rather than customize the collation here, I've opted to match the framework's CI tests and specify `10`, which would give us MariaDB 10.6.